source/renderer/app/config/walletsConfig.js

```diff
@@ -40,7 +40,7 @@ export const IS_AUTOMATIC_WALLET_MIGRATION_ENABLED = false;
 // Wallet assets feature toggle enable/disable
 export const WALLET_ASSETS_ENABLED = true;
 
-export const IS_WALLET_ASSETS_AMOUNT_FORMATTING_ENABLED = false;
+export const IS_WALLET_ASSETS_AMOUNT_FORMATTING_ENABLED = true;
```

Comment:
@daniloprates can we please get rid of this constant all together?

Reply:
✔️